### DISCUSSION TITLE
I need guidance.

### DISCUSSION THREAD

**Discussion by DiegoteWhite** [supabase/supabase#20219] [Questions] [answered]
I'm using FlutterFlow with Supabase.  I need guidance to make this base perform a function.  When a record is added to FF, it disables a series of validations to be carried out on the back end and triggers the addition of a few records to a table.  Where do I have to go: API, functions... I would appreciate some guidance for someone who wants to learn but doesn't have much idea.

**Comment by dnlmgwi** [comment]
Have you enable RLS, Row Level Security on the table you adding records to?

I would suggest you start here:
1. https://supabase.com/docs/learn/auth-deep-dive/auth-row-level-security

2. https://supabase.com/docs/learn/auth-deep-dive/auth-policies

Be sure that you are using the ANON_KEY because the SERVICE_ROLE has the ability to bypass Row Level Security.

**Reply by DiegoteWhite** [reply]
Thanks for the answer, but I don't think it's for me.

**Comment by awalias** [comment, marked as answer]
If you prefer writing Javascript instead of SQL I would personally recommend [edge functions](https://supabase.com/docs/guides/functions) to do your backend logic here.

For people more advanced with SQL, database triggers can work, but it's more advanced.
